time/build.gradle.kts

```diff
@@ -40,6 +40,7 @@ dependencies {
     compileOnly(AutoService.annotations)
 
     api("io.spine:spine-base:$baseVersion")
+    implementation("io.spine:spine-validate:$baseVersion")
```

Comment:
I am not sure I understand this properly.

`time` library provides its own Protobuf options. So it looks there should be an implementation of `OptionsProvider` (publishing `time_options.proto` to the extension registry) which would be auto-discovered by the mechanism established in recent `base`.

If that's not the case, I don't understand why we need this dependency in `time`. 

Let's discuss vocally if needed.

Reply:
You're right. I was in a rush... :)